website/source/docs/cluster/bootstrapping.html.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: "docs"
+page_title: "Creating a cluster"
+sidebar_current: "docs-cluster-bootstrap"
+description: |-
+  Learn how to bootstrap a Nomad cluster.
+---
+
+# Creating a cluster
+
+Nomad clusters in production comprises of a few Nomad servers, clients and
+optionally Consul servers and clients.Before we start discussing the specifics
+around bootstrapping clusters we should discuss the network topology. Nomad
+models infrastructure as regions and datacenters. Regions may contain multiple
+datacenters. Servers are assigned to a single region and hence a region is a
+single scheduling domain in Nomad. Multiple regions can be connected together
```

Comment:
`Multiple regions can be connected together` -> `Each cluster of Nomad Servers support only one region, however multiple Regions can be stitched together to allow a globally coherent view of an organization's resources.`  Or something like that.